ci/scripts/pullFabricImages.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash -e
 set -euo pipefail
 
-# FABRIC_VERSION is set in ci/azure-pipelines.yml
+FABRIC_VERSION=${FABRIC_VERSION:-2.2}
```

Comment:
Doesn't have to be addressed in this PR, but just curious if there is a way to test against the latest stable Fabric instead of hardcoding the version in several files, as we'll undoubtedly forget to update all the version numbers.

Reply:
There is. I'm only leaving it at 2.2 until we tag the 2.2.0 samples and then master can migrate to 2.3

Reply:
Since 2.2 is LTS I wanted to explicitly continue testing against 2.2 until we tag